tests/pyfunc/test_model_export.py

```diff
@@ -126,3 +194,51 @@ def test_cli_predict(self):
             result_df = pandas.read_csv(output_csv_path, header=None)
             np.testing.assert_array_equal(result_df.values.transpose()[0],
                                           self._knn.predict(self._X))
+
+    def _cli_predict_with_conda_env(self, extra_args):
+        with TempDir() as tmp:
+            model_path = tmp.path("knn.pkl")
+            with open(model_path, "wb") as f:
+                pickle.dump(self._knn, f)
+
+            # create a conda yaml that installs mlflow from source in-place mode
+            conda_env_path = tmp.path("conda.yml")
+            with open(conda_env_path, "w") as f:
```

Comment:
Could we extract this conda-writing logic to a private method (that takes the `tmp` as its input), to avoid duplicating it?